sig-network/README.md

```diff
@@ -42,6 +42,12 @@ The Chairs of the SIG run operations and processes governing the SIG.
 * Shane Utt (**[@shaneutt](https://github.com/shaneutt)**), Kong
 * Tim Hockin (**[@thockin](https://github.com/thockin)**), Google
 
+### Technical Leads
+The Technical Leads of the SIG establish new subprojects, decommission existing
+subprojects, and resolve cross-subproject technical issues and decisions.
+
+* Dan Winship (**[@danwinship](https://github.com/danwinship)**), Red Hat
```

Comment:
If we're adding this section, I suppose we should put me in there to be explicit.

Reply:
sg 24be06e84480496e6c281c95bc2f7c63cc142fa8 :+1: